frontend/actions/siteActions.js

```diff
@@ -109,4 +113,26 @@ export default {
       })
       .catch(alertError);
   },
+
+  removeBasicAuthFromSite(siteId) {
+    return federalist.removeBasicAuthFromSite(siteId)
+      .then((site) => {
+        if (site) {
+          dispatchSiteBasicAuthRemovedAction(site);
+        }
+      })
+      .then(() => alertActions.alertSuccess('Successfully removed basic authentication.'))
+      .catch(alertError);
+  },
+
+  saveBasicAuthToSite(siteId, basicAuth) {
+    return federalist.saveBasicAuthToSite(siteId, basicAuth)
+      .then(site => {
+        if (site) {
+          dispatchSiteBasicAuthSavedAction(site);
+        }
+      })
+      .then(() => alertActions.alertSuccess('Successfully added basic authentcation.'))
```

Comment:
```suggestion
      .then(() => alertActions.alertSuccess('Successfully added basic authentication.'))
```

Reply:
@davemcorwin  - good point. what about instead of "Disable" we change the button to "Delete" or "Remove"?